memor/session.py

```diff
@@ -103,69 +89,54 @@ def __add__(self, other_object):
             return Session(messages=new_messages)
         raise TypeError(UNSUPPORTED_OPERAND_ERROR_MESSAGE.format("+", "Session", type(other_object).__name__))
 
-    def __radd__(self, other_object):
+    def __radd__(self, other_object: Any) -> Any:
```

Comment:
`def __radd__(self, other_object: Any) -> Session`

Reply:
Since it's in the same class it can not be used.